packages/account/src/Sections/Security/LoginHistory/login-history-content.tsx

```diff
@@ -23,10 +24,9 @@ type TLoginHistoryContent = {
 };
 
 const LoginHistoryContent = observer(({ data }: TLoginHistoryContent) => {
-    const { ui } = useStore();
-    const { is_mobile } = ui;
+    const { isDesktop } = useDevice();
```

Comment:
since we removed mobx store dependency, we can remove observer wrapper as well. can you check this?

Reply:
Sure 🫡